exercises/concept/calculator-conundrum/.meta/config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "blurb": "Learn about exception-handling by making a simple calculator.",
+  "authors": [
+    "rv02",
+    "jmrunkle"
+  ],
+  "files": {
```

Comment:
We need to add an "editor" stanza with the IllegalOperationException and the SimpleOperation java files.